### ISSUE TITLE
run() method broken in PHP 7.4.13

### ISSUE THREAD

**Issue by webclaus** [paragonie/easydb#132]
Hi guys,

After upgrading from PHP 7.4.12 to 7.4.13 I get errors on simple delete queries when using the run() method.

For example:

$easydb->run('delete from something where id = ? limit 1', 1234);

Will throw an exception:

```
PHP Fatal error:  Uncaught PDOException: SQLSTATE[HY000]: General error: 2014 Cannot execute queries while other unbuffered queries are active.  Consider using PDOStatement::fetchAll().  Alternatively, if your code is only ever going to run against mysql, you may enable query buffering by setting the PDO::MYSQL_ATTR_USE_BUFFERED_QUERY attribute. in .../vendor/paragonie/easydb/src/EasyDB.php:1310
Stack trace:
#0 .../vendor/paragonie/easydb/src/EasyDB.php(1310): PDOStatement->fetchAll(2)
#1 .../vendor/paragonie/easydb/src/EasyDB.php(1009): ParagonIE\EasyDB\EasyDB->getResultsStrictTyped(Object(PDOStatement), 2)
#2 .../vendor/paragonie/easydb/src/EasyDB.php(948): ParagonIE\EasyDB\EasyDB->safeQuery('delete from cli...', Array, 2, false, true)
[...]
```
I can get around this by avoiding the getResultsStrictTyped() call, using safeQuery directly and asking for numRowsAffected, but that seems rather cumbersome for a simple task like this :)

**Comment by orjanalmen** [comment]
Same for me. Worked fine and then after an php upgrade it started to do these errors. Downgrading and it works fine again.

**Comment by usarise** [comment]
@webclaus This bug is fixed in the php-7.4.14RC1

**Comment by webclaus** [comment]
Aaah, so a PHP bug, really.

Well, perfect, we'll lean back and wait for the 7.4.14 release 👍
